hypervisor/include/arch/x86/asm/tsc.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2018 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef ARCH_X86_TSC_H
+#define ARCH_X86_TSC_H
+
+#include <types.h>
+
+#define TSC_PER_MS	((uint64_t)get_tsc_khz())
+
+/**
+ * @brief read tsc.
+ *
+ * @return tsc value
+ */
+static inline uint64_t rdtsc(void)
+{
+	uint32_t lo, hi;
+
+	asm volatile("rdtsc" : "=a" (lo), "=d" (hi));
+	return ((uint64_t)hi << 32U) | lo;
+}
+
+/**
+ * @brief  Get tsc.
```

Comment:
Same comment, can you be more verbose. What is the difference between `read tsc` and `get tsc` for example? I kind of guess that it's mainly about the unit used for the return value, is that it? In any case, more verbose text for the `@brief` and `@return` strings should make it immediately clear what the difference is (and what the function is about).